whatlies/embeddingset.py

```diff
@@ -446,6 +446,38 @@ def merge(self, other):
         """
         return EmbeddingSet({**self.embeddings, **other.embeddings})
 
+    def assign(self, **kwargs):
+        """
+        Adds properties to every embedding in the set based on the keyword arguments.
+
+        This is very useful for plotting because a property can be used to assign colors. This method is very
+        similar to `.add_property` but it might be more convenient when you want to assign multiple properties
+        in one single statement.
+
+        Arguments:
+            kwargs: (name, func)-pairs that describe the name of the property as well as a function on how to calculate it
```

Comment:
I am not sure if it should be mentioned that the `func` is a function **that takes an `Embedding` instance as input**?!

Reply:
Explicit > Implicit.